plugins/woocommerce/includes/wc-account-functions.php

```diff
@@ -164,6 +164,33 @@ function wc_get_account_menu_item_classes( $endpoint ) {
 	return implode( ' ', array_map( 'sanitize_html_class', $classes ) );
 }
 
+/**
+ * Get current page in order to set the current menu item.
+ *
+ * @param string $endpoint Endpoint.
+ * @return string
+ * @since 9.2.0
+ */
+function wc_get_account_menu_aria_current( $endpoint ) {
```

Comment:
The bulk of the "current" page detection code is duplicated from `wc_get_account_menu_item_classes`. Why don't we move this to a helper function? This will ensure the two never become out of sync and they behave in the same way.

i.e. (if we're sticking with functions) `wc_is_current_account_menu_item( $endpoint )` which returns boolean, then consume from both `wc_get_account_menu_aria_current` and `wc_get_account_menu_item_classes`.

Reply:
Or use directly in the HTML:

```php
<?php echo wc_is_current_account_menu_item( $endpoint ) ? 'aria-current="page"' : '';  ?>
```